Core/HttpHandlerProxy.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Security.Authentication;
+using System.Text;
+using Microsoft.Net.Http.Headers;
+using System.Threading.Tasks;
+using System.Threading;
+using Microsoft.Exchange.WebServices.Data;
+using Microsoft.Exchange.WebServices.NETStandard.Extensions;
+
+namespace Microsoft.Exchange.WebServices.NETStandard.Core
+{
+    internal class HttpHandlerProxy : DelegatingHandler

```

Comment:
Implement IDisposable to correctly release handlers, as you may be disposed before initializing InnerHandler

Reply:
agree